contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,317 @@
+
+They also offer product analytics, so you don't need a separate platform or integration to link your analytics data to your A/B test results.
+
+### Which platforms does VWO support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **Flutter:** <span className="text-green text-lg">✔</span>
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does VWO cost?
+
+VWO mobile app A/B tests starts at $1,526 per month for up to 10k users. This is billed annually, so it means an upfront cost of $18,312. Also note that this price is only for mobile A/B tests. If you want to include analytics in your package, prices start at an additional $15,936 per year.
+
+Lastly, if you also want to run server-side tests, you also need to purchases a separate package. Prices for this start at $28,164 per year.
+
+## 3. Optimizely
+
+- **Free tier:** <span className="text-red text-lg">✖</span>
+- **Open source:** <span className="text-red text-lg">✖</span>
+- **Transparent pricing:** <span className="text-red text-lg">✖</span>
+- **Includes product analytics:** <span className="text-red text-lg">✖</span>
+- **Get started without talking to sales:** <span className="text-red text-lg">✖</span>
+- **Best for:** Marketing teams
+
+![Optimizely mobile app A/B tests](../images/blog/best-mobile-app-ab-testing-tools/optimizely-testing.png)
+
+### What is Optimizely?
+
+[Optimizely](/blog/posthog-vs-optimizely) is an all-in-one system for marketing that includes A/B testing **and** more – like content management, campaign planning, asset management, and checkout customizations. 
+
+It's built for marketers in large enterprises looking to optimize their content, apps, and e-commerce experiences.
+
+### Which platforms does Optimizely support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **Flutter:** <span className="text-green text-lg">✔</span>
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does Optimizely cost?
+
+Optimizely's pricing is sales-driven and completely opaque, so it's not clear how much it costs. However, according to [Splitbase](https://splitbase.com/blog/optimizely-pricing), Optimizely will cost you at least $36,000 per year.
+
+## 4. Firebase
+
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-red text-lg">✖</span> 
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-red text-lg">✖</span>
```

Comment:
If Firebase requires Google Analytics, isn't this the equivalent of having product analytics?